### ISSUE TITLE
Error Message does not clear line

### ISSUE THREAD

**Issue by tessprime** [iarna/xenforo-to-epub#2]
If you attempt to fetch from an invalid url you get the following error:

`An error occured: Cannot read property 'replace' of nullwn.org/works/1234567890`

What appears to be happening is that the progress bar resets to the beginning of the line after writing to the console, resulting in the error message overwriting the progress bar text.

**Comment by iarna** [comment]
If you pass in `--debug` it will give you a full stack trace, which is helpful when tracking down root causes…
```
TypeError: Cannot read property 'replace' of nulleofourown.org/works/234928423874382
    at filenameize (/Users/rebecca/code/fetch-fic/util/filenameize.js:5:13)
    at /Users/rebecca/code/fetch-fic/ff-get.js:72:22
From previous event:
    at _reallyRead (/Users/rebecca/code/fetch-fic/ff-get.js:70:78)
    at /Users/rebecca/code/fetch-fic/ff-get.js:10:12
    at processImmediate [as _immediateCallback] (timers.js:383:17)
From previous event:
    at read (/Users/rebecca/code/fetch-fic/ff-get.js:7:28)
    at Object.<anonymous> (/Users/rebecca/code/fetch-fic/ff.js:159:17)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:974:3
```

What's particularly interesting to me here is that some site handlers (eg, xenforo) give errors that make a bit more sense:
```
An error occured: Got status: 404 Not Found for https://forums.spacebattles.com/threads/asldfj.2394327438/threadmarks
```

But others (like ao3 and wikipedia) give the error you shared. And worst of all, `ffnet` writes an empty file named `.fic.toml`.

**Comment by iarna** [comment]
It turns out ffnet returns a 200 when it does find a fic. Because of course it does.

**Comment by iarna** [comment]
I fixed this in v3.4.1. Thank you for reporting it!
